main.go

```diff
@@ -174,6 +195,7 @@ func GetStaticConfigs(svc kafkaClient, opt_filter ...regexp.Regexp) ([]Prometheu
 }
 
 func main() {
+	flag.Var(&tagFilters, "tags", "A key=value for filtering by tags. Flag can be specified multiple times.")
```

Comment:
The flag should maybe be called `tag` instead if you have to specify it multiple times for multiple flags rather than just something like `-tags a=b,c=d`